builtin/providers/google/resource_sql_database.go

```diff
@@ -67,6 +68,7 @@ func resourceSqlDatabaseCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	err = sqladminOperationWait(config, op, "Insert Database")
+	googleMutexKV.Unlock(instanceMutexKey(project, instance_name))
```

Comment:
I'd rather see this after line 60 in a `defer`, to be honest, just to cover our bases.